src/store/reducers/auth/index.js

```diff
@@ -1,10 +1,14 @@
 import { GET_AUTH } from "../../types";
 
 const INITIAL_STATE = {
-    data: {}
+    data: {
+        authAddress: null,
+        accountType: null
+    }
```

Comment:
Why `data` is necessary?
Why `authAddress` and `accountType` should not be inside the state directly?